From 782e5a75e961ccd3f508d20290b04542988f5265 Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@nxp.com>
Date: Tue, 18 Jul 2017 17:27:02 +0800
Subject: [PATCH 2149/5242] MLK-15973 ARM: dts: imx6sx-sabreauto: correct CD
 pin.

commit  4cdaa0a9502a531fcc9f8d2433d0fba1a3c216bc from
https://source.codeaurora.org/external/imx/linux-imx.git

Correct the CD pin for baseboard SD slot, otherwise
the card detection can't work.

Signed-off-by: Haibo Chen <haibo.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sx-sabreauto.dts |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx-sabreauto.dts b/arch/arm/boot/dts/imx6sx-sabreauto.dts
index 5f7cde2..5fee360 100644
--- a/arch/arm/boot/dts/imx6sx-sabreauto.dts
+++ b/arch/arm/boot/dts/imx6sx-sabreauto.dts
@@ -995,8 +995,7 @@
 			MX6SX_PAD_SD4_DATA1__USDHC4_DATA1	0x17071
 			MX6SX_PAD_SD4_DATA2__USDHC4_DATA2	0x17071
 			MX6SX_PAD_SD4_DATA3__USDHC4_DATA3	0x17071
-			MX6SX_PAD_SD4_DATA7__GPIO6_IO_21	0x17071 /* CD */
-			MX6SX_PAD_SD4_DATA6__GPIO6_IO_20	0x17071 /* WP */
+			MX6SX_PAD_USB_H_STROBE__GPIO7_IO_11	0x17071 /* CD */
 		>;
 	};
 
-- 
1.7.9.5

